### PR TITLE
Fix unicode error if non-latin email template used

### PR DESCRIPTION
**Pull request by toropok**
## Description of the issue/feature this PR addresses

If email template contains non latin chars - `emailview.py` fails on the line 514 when tries to get translation of the particular template.

## Current behavior before PR

fails to get translation of the email template

## Desired behavior after PR is merged

do not fail

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: src/bika/lims/browser/publish/emailview.py [hunk 1]
```diff
@@ -511,14 +511,14 @@ def render_email_template(self, template, template_context=None):
         """
 
         # allow to add translation for initial template
-        template = self.context.translate(_(template))
+        template = self.context.translate(_(safe_unicode(template)))
         recipients = self.email_recipients_and_responsibles
         if template_context is None:
             template_context = {
-                "recipients": "<br/>".join(recipients),
+                "recipients": "<br/>".join(map(safe_unicode, recipients)),
```
Comment (ramonski) [inline src/bika/lims/browser/publish/emailview.py]: Please provide the joining element `"<br/>"` also as unicode `u"<br/>"`

**Review by ramonski — CHANGES_REQUESTED** [review]
Thanks Leo! Just a small thing to adapt, then we can merge it.

File: src/bika/lims/browser/publish/emailview.py (hunk 1 above)
Comment (toropok) [reply src/bika/lims/browser/publish/emailview.py]: my bad.. corrected

**Review by ramonski — APPROVED** [review]
Thanks!

**Pull request merged**